edb/lib/cfg.edgeql

```diff
@@ -192,6 +192,19 @@ ALTER TYPE cfg::AbstractConfig {
             'Whether inserts are allowed to set the \'id\' property.';
     };
 
+    CREATE PROPERTY simple_scoping -> std::bool {
+        CREATE ANNOTATION cfg::affects_compilation := 'true';
+        CREATE ANNOTATION std::description :=
+            'Whether to use the new simple scoping behavior \
+            (disable path factoring)';
+    };
+
+    CREATE PROPERTY warn_old_scoping -> std::bool {
+        CREATE ANNOTATION cfg::affects_compilation := 'true';
+        CREATE ANNOTATION std::description :=
+            'Whether to warn when depending on old scoping behavior.';
+    };
```

Comment:
What's the default value here? False? Should we have this be True?

Reply:
The default value is not having a value, and falling back to the presence or absence of the future.
(This is kind of tricky; there is a table in the RFC)